torch/_inductor/mkldnn_lowerings.py

```diff
@@ -544,24 +547,79 @@ def qlinear_unary(
             bias_dtype = None if bias is None else bias.get_dtype()
 
             choices: List[ChoiceCaller] = []
+
             if use_max_autotune():
+                # NOTE: since vector weight scale is supported with oneDNN qlinear in PyTorch,
+                # we are able to support the case of per-token quantized activation & per-channel quantized weights
+                # if x_zp is not present in a pattern, since in that case, we can apply the actual scale of x
+                # after oneDNN qlinear in a replacement pattern. This is being done for a smoothquant pattern.
+                # Inductor codegen is able to make it an epilogue fusion.
+                # We don't support the case of per-token quantized activation with oneDNN qlinear if
+                # the activation is quantized asymmetrically, or if the per-token quantized activation is
+                # quantized symmetrically but still has a corresponding zero-points tensor,
+                # which would be all zeros in case of int8).
```

Comment:
Why do we need to mention this?

Reply:
Documentation of unsupported cases - since auto-tuning has two candidates - the GEMM template is forced to support a subset of what it can support, since oneDNN qlinear doesn't support all cases

Reply:
On second thought, I should probably move this comment block before `if use_max_autotune():`